### ISSUE TITLE
Using wrong npm error variable

### ISSUE THREAD

**Issue by cb1kenobi** [PRINTR3D/formide-client#98]
Noticed this while looking for programmatic examples of using npm and noticed when an npm call fails, you're passing the wrong error variable. There's a handful of places that have the same issue.

```
npm.commands.install([packageName], function (installErr, data) {
    if (installErr) return cb(err); // <----------  should be installErr
```

https://github.com/PRINTR3D/formide-client/blob/master/core/modules/update/index.js#L65
https://github.com/PRINTR3D/formide-client/blob/master/core/modules/update/index.js#L87
https://github.com/PRINTR3D/formide-client/blob/master/core/modules/update/index.js#L105
https://github.com/PRINTR3D/formide-client/blob/master/core/modules/update/index.js#L122
https://github.com/PRINTR3D/formide-client/blob/master/core/modules/update/index.js#L141

**Comment by ChrisTerBeke** [comment]
Thanks! Will push a fix in the next release

**Comment by ChrisTerBeke** [comment]
Fixed in npm pushed version 0.2.33
